### ISSUE TITLE
Registration page not enforcing the input of a 1. first name 2. last name, and 3. username correctly

### ISSUE THREAD

**Issue by guillempuche** [lightningrodlabs/acorn#157]
Version: 1.1.0-alpha (1.1.0-alpha)
I haven't filled the parameter `Last name`.

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/3519924/188969898-b8ce66f3-9870-40e1-b182-1634d712c170.png">

Even if I restart the app, it doesn't clean the data itself. I can't do anything.

Stack trace:

```
Cannot read property 'toUpperCase' of undefined

TypeError: Cannot read property 'toUpperCase' of undefined
    at Zu (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:313560)
    at Qa (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:57951)
    at ws (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:104333)
    at xc (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:96859)
    at _c (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:96784)
    at fc (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:93807)
    at file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:45339
    at t.unstable_runWithPriority (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:317:3844)
    at Ko (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:45048)
    at Xo (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:45284)
```

**Comment by Connoropolous** [comment]
thanks for reporting. 

You may be able to clear your state, and start over. 

I just drafted a new page on how to do this, please check it out and see if it helps: 
https://docs.acorn.software/troubleshooting/clearing-application-data

We will try to get a fix in to the next release to prevent this from happening again!

**Comment by Connoropolous** [comment]
It looks like this is one of a few minor issues involving the profile registration page, so I just changed the title to reflect that. 

@pegahvaezi and @weswalla let's make sure we fix this and get it in with the next release

as a note to self, I find that the validation rules in the `ProfileEditForm.js` are not implemented inside of a `useEffect` which they should be.

**Comment by Connoropolous** [comment]
This is fixed in v2.0.0, out now.
